packages/browser-lib/README.md

```diff
@@ -0,0 +1,28 @@
+# `@sa11y/browser-lib`
```

Comment:
Not sure if `browser-lib` is an apt name for the library that produces sa11y.min.js. 
Suggestions welcome!